detox/test/src/Screens/SyncScreen.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+import {
+  Text,
+  TouchableOpacity,
+  ActivityIndicator,
+  Platform,
+  StyleSheet,
+  SafeAreaView,
+  View,
+} from 'react-native';
+
+const styles = StyleSheet.create({
+  screen: {
+    flex: 1,
+  },
+
+  container: {
+    justifyContent: 'flex-start',
+    paddingTop: Platform.OS === 'ios' ? 30 : 0,
+  },
+
+  textButton: {
+    color: 'blue',
+    marginVertical: 10,
+    textAlign: 'center',
+  },
+});
+
+export default class SyncScreen extends Component {
+  state = {
+    showIndicator: false,
+  }
+
+  render() {
+    return (
+      <SafeAreaView style={styles.container}>
+        <View testID="SyncScreenContainer" style={styles.container}>
+          <TouchableOpacity testID="showIndicator" onPress={() => this.setState({ showIndicator: true })}>
+            <Text style={styles.textButton}>Show Activity Indicator</Text>
+          </TouchableOpacity>
+
+          { this.state.showIndicator && <ActivityIndicator testID="indicator" key="indicator" size="large" color="black" /> }
```

Comment:
This test is invalid and will never be idle. If an indicator appears and animates, the system is considered busy, and will never be idle.